PoW-vs-Pos/en/en_transcription.md

```diff
@@ -0,0 +1,102 @@
+
+(15:30) unhealthy they're not really crewing that value to shares overall and so i think when it comes down to kind of what is trying to be money the thing that does almost nothing other than just be money is the best money and something that tries to do multiple things including proof of stake as one of the components inherently gives it less monetary premium in a proof of work system power is distributed between miners developers and individual nodes your ability to be a miner is based on your ability to put forth money and find low-cost
+
+(16:02) electricity rather than the original miners having an advantage and increasing their advantage over time new miners have a technical advantage over existing miners because they buy newer machines with more processing power if you if you bought a mining rig four years ago and i bought one right now i have it i have a bigger advantage or i can capture more coins that's rare that the new entrant has an advantage over the old entrant right so like you you came into it four years ago it's harder for you to capture and
+
+(16:35) mine the coins and it is for me and i think that what that does and when you look at proof of stake it's a little bit different if you had first access to the coins you now have an advantage over somebody who's now just coming in that's trying to capture them and employ them to the network and so from this system bitcoin mining businesses old and new are all in constant competition with each other this is opposite to proof of stake where there is no competition once you have ownership of the tokens you can keep
+
+(17:07) that power in the system forever plus bitcoin developers went to great lengths to make it easy and cheap to run a full node which allows any user to audit the entire blockchain and reject blocks that don't conform with the rules of the network as a result the real power rests with the nodes rather than the miners if miners try to collude and mine invalid blocks the node network simply rejects those blocks so this system is similar to the us constitution where it has three branches of government you would
+
+(17:38) have the executive branch the legislative branch and the judicial branch these branches would have the ability to overrule each other in certain contexts and in addition to this overruling they would have staggered term limits so this is by design you wouldn't want political people to have power forever you would want to vote them out of office in favor of newer people with new ideas and this is how the us works so like the united states the bitcoin network has three branches that limit each other's power the node network has
```

Comment:
Distinguere le frasi non del narratore `so this system is similar to the us constitution where it has three branches of government you would [...] you would want to vote them out of office in favor of newer people with new ideas and this is how the us works`